.github/workflows/upload-python-package-to-codeartifcat.yml

```diff
@@ -3,7 +3,6 @@ on:
   push:
     branches:
       - main
-      - feature/**
```

Comment:
I'm not sure we should remove this as it permits verification builds on feature branches (without pushing to CodeArtifact).

If this is to be removed, the `if` logic below (that prevents pushes to CodeArtifact when building on feature branches, but not the main branch) should be removed too.